agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaSchedule.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <v-calendar
+      ref="calendar"
+      v-model="focus"
+      color="primary"
+      :events="eventList"
+      :event-color="getEventColor"
+      :type="type"
+      @click:event="showEvent"
+      @click:more="viewDay"
+      @click:date="viewDay" />
+    <agenda-event />
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    eventList: {
+      type: Array,
+      default: () => null
+    }
+  },
+  data: () => ({
+    focus: '',
+    type: 'week'
+  }),
+  mounted() {
+    this.$root.$on('switch-type', (data) => {
+      this.type = data;
+    });
+  },
+  methods:{
+    getEventColor (event) {
+      return event.color;
+    },
+    showEvent ({ nativeEvent, event }) {
+      const open = () => {
```

Comment:
no need to define the function here, you can use it in `else`

Reply:
it will be done in new US